pkg/e2sub/subscriptions.go

```diff
@@ -135,7 +136,7 @@ func runAddSubscriptionCommand(cmd *cobra.Command, args []string) error {
 	if e2NodeID == "" {
 		return errors.New("e2NodeID must be specified with --e2NodeID")
 	}
-	smID, _ := cmd.Flags().GetString("smId")
+	smIDString, _ := cmd.Flags().GetString("smId")
```

Comment:
Can we define two separate flags? Service model name and service model version. It is much cleaner. We don't need to concatenate these two which was done previously.

Reply:
The user should not figure out how to define an ID and what is the delimiter between name and version